src/main/resources/env.schema.json

```diff
@@ -1,120 +1,120 @@
 {
-  "$schema": "http://json-schema.org/draft-04/schema#",
+  "$schema": "http://json-schema.org/draft-07/schema#",
   "title": "OTP Middleware JSON schema defined to validate the environment configuration.",
-  "description": "Note: Required values must be defined under the 'required' array at the bottom of this schema. Default values are examples only and are not applied at runtime.",
+  "description": "Note: Required values must be defined under the 'required' array at the bottom of this schema.",
   "type": "object",
   "properties": {
     "AUTH0_API_CLIENT": {
       "type": "string",
-      "default": "test-auth0-client-id",
+      "examples": ["test-auth0-client-id"],
       "description": "API client id required to authenticate with Auth0."
     },
     "AUTH0_API_SECRET": {
       "type": "string",
-      "default": "test-auth0-secret",
+      "examples": ["test-auth0-secret"],
       "description": "API secret id required to authenticate with Auth0."
     },
     "AUTH0_DOMAIN": {
       "type": "string",
-      "default": "test.auth0.com",
+      "examples": ["test.auth0.com"],
       "description": "Auth0 tenant URL."
     },
     "AWS_PROFILE": {
       "type": "string",
-      "default": "default",
+      "examples": ["default"],
       "description": "AWS profile for credentials"
     },
     "AWS_API_SERVER": {
       "type": "string",
-      "default": "aws-api-id.execute-api.us-east-1.amazonaws.com",
+      "examples": ["aws-api-id.execute-api.us-east-1.amazonaws.com"],
       "description": "For generating the swagger document at runtime. Can be null, however that will prevent tools such as swagger-UI from submitting test requests to the API server."
     },
     "AWS_API_STAGE": {
       "type": "string",
-      "default": "stage-name",
+      "examples": ["stage-name"],
       "description": "For generating the swagger document at runtime. Can be null, however that will prevent tools such as swagger-UI from submitting test requests to the API server."
     },
     "BUGSNAG_API_KEY": {
       "type": "string",
-      "default": "bugsnag-api-key",
+      "examples": ["bugsnag-api-key"],
       "description": "A valid Bugsnag authorization token."
     },
     "BUGSNAG_EVENT_JOB_DELAY_IN_MINUTES": {
       "type": "integer",
-      "default": "1",
+      "examples": ["1"],
       "description": "Bugsnag event job frequency."
     },
     "BUGSNAG_EVENT_REQUEST_JOB_DELAY_IN_MINUTES": {
       "type": "integer",
-      "default": "5",
+      "examples": ["5"],
       "description": "Bugsnag event request frequency."
     },
     "BUGSNAG_ORGANIZATION": {
       "type": "string",
-      "default": "bugsnag-organization-id",
+      "examples": ["bugsnag-organization-id"],
```

Comment:
Might be good to use a sample that looks like the ID needed. E.g., `123e4567e89b12d3a4564266`